tests/test_io.py

```diff
@@ -1108,6 +1108,13 @@ def test_read_excel_dtypes():
         dtypes = {"a": np.int64, "b": str, "c": np.float64}
         check(assert_type(read_excel(path, dtype=dtypes), pd.DataFrame), pd.DataFrame)
 
+def test_excel_reader():
```

Comment:
pre-commit is failing here because you need an extra blank line.

Do `pre-commit install` so that the code checks (`black`, etc.) run before pushing.

Reply:
Sorry about that, totally forgot about pre-commit (it gives me troubles with ruff on nixos, I start to question my OS choice :sweat_smile:)